src/main/java/com/example/javacohort3/ZipCodeBank/controllers/AccountController.java

```diff
@@ -44,19 +42,22 @@
         Account account = accountService.getAccountById(accountId);
 
         log.info("[GET] " + account);
-        return new ResponseEntity<>(account, HttpStatus.OK);
+        return new ResponseEntity<>(new ResponseDetails(HttpStatus.OK, "Success", account), HttpStatus.OK);
     }
 
     // Get accounts by customer ID
     @RequestMapping("/customers/{customerId}/accounts")
     public ResponseEntity<?> getAllAccountsByCustomerId(@PathVariable Long customerId) {
+
+        customerService.verifyCustomer(customerId);
         accountService.verifyCustomerById(customerId);
```

Comment:
You need to use the same logic we discoverd yesterday as far as using the lamdas  & removeif method

Reply:
Actually sorry I thought this was the service but get rid of extra verify method